<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="setupFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>401</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="numElevatorsLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevators</string>
     </property>
    </widget>
    <widget class="QLabel" name="numFloorsLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Floors</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="numElevatorsSpin">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>45</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="numFloorsSpin">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>45</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="safeFloorLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Safe Floor</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="safeFloorSpin">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>45</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QFrame" name="settingsTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="settingsTitleLabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Building Settings</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="rebuildButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>(Re)start using settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fireAlarmButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger fire alarm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="outageButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Create power outage</string>
     </property>
    </widget>
    <zorder>numElevatorsLabel</zorder>
    <zorder>numFloorsLabel</zorder>
    <zorder>numElevatorsSpin</zorder>
    <zorder>numFloorsSpin</zorder>
    <zorder>safeFloorLabel</zorder>
    <zorder>safeFloorSpin</zorder>
    <zorder>rebuildButton</zorder>
    <zorder>fireAlarmButton</zorder>
    <zorder>outageButton</zorder>
    <zorder>settingsTitle</zorder>
   </widget>
   <widget class="QFrame" name="elevatorStatusFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>361</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="elevatorStatusTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="elevatorStatusTitleLabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Elevator Status</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>321</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="elevatorStatusLayout"/>
    </widget>
   </widget>
   <widget class="QFrame" name="currentElevatorFrame">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>201</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="currentElevatorTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="currentElevatorLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Current Elevator</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="floorPanelTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="floorPanelLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Floor Panel</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="elevatorSelectionLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevator#:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="elevatorSelectionSpin">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>45</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
    </widget>
    <widget class="QPushButton" name="passengerEnterButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Passenger +1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="passengerExitButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Passenger -1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="directionUp">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="fromFloorLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>From floor:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="fromFloorSpin">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>45</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
    </widget>
    <widget class="QPushButton" name="directionDown">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="controlPanelFrame">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>201</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="controlPanelTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="controlPanelLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Control Panel</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="safetyTestTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="safetyTestLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Safety Feature Test</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="testOverload">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Overload</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="testObstacle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Door obstacle</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="toggleBuildingSafety">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Building safety response</string>
     </property>
    </widget>
    <widget class="QPushButton" name="doorOpenButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;|&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="doorCloseButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;|&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="localFireAlarmButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Test local fire alarm</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>160</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="controlPanelLayout"/>
    </widget>
   </widget>
   <widget class="QFrame" name="eventLogFrame">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>401</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="eventLog">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>401</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; background-color: black;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="eventLogTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="eventLogTitleLabel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Event Log</string>
      </property>
     </widget>
    </widget>
    <zorder>eventLogTitle</zorder>
    <zorder>eventLog</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
